lib/timers.js

```diff
@@ -83,10 +83,19 @@ function listOnTimeout() {
   debug('timeout callback ' + msecs);
 
   var now = Timer.now();
-  debug('now: ' + now);
+  debug('now: %d', now);
```

Comment:
@saquibkhan `now` is a non negative 64 bits integer number that is stored as a double if its value is larger than a V8 SMI. Plus the `%d` format supports non-integer numbers since it uses the function `Number` to convert the parameter corresponding to this format to a number.